HTTPClient/Source/HTTPClientError.swift

```diff
@@ -4,6 +4,19 @@
 //
 
 import Foundation
+import Alamofire
+
+extension AFError: NetworkErrorConvertible {
+    func networkError() -> NetworkError {
+        if let invalidCode = responseCode {
```

Comment:
why no check for type?

Reply:
responseCode - /// The response code of a `.responseValidationFailed` error.
so i don't see any sense to unwrap type to get the same value